app/konnect/org-management/audit-logging/reference.md

```diff
@@ -5,10 +5,38 @@ content_type: reference
 beta: true
 ---
 
-{{site.konnect_short_name}} logs events in [ArcSight CEF Format](https://docs.centrify.com/Content/IntegrationContent/SIEM/arcsight-cef/arcsight-cef-format.htm).
+## Log overview
+{{site.konnect_short_name}} captures three types of events:
+
+* Authentication: triggered when a user attempts to log into the Konnect web application or use the Konnect API via a personal access token. Also triggered when a system account access token is used.
+* Authorization: triggered when a permission check is made for a user or system account against a resource.
+* Access logs: triggered when a request is made to the Konnect API.
+
+## Log format
+
+{{site.konnect_short_name}} delivers log events in [ArcSight CEF Format](https://docs.centrify.com/Content/IntegrationContent/SIEM/arcsight-cef/arcsight-cef-format.htm). Webhook calls include a batch of events; each event is formatted in CEF and separated by a newline. The `Content-Type` is `application/text`.
+
+To minimize payload size, the message body is compressed.  The `Content-Encoding` is `application/gzip`.
+
+All log entries include the following attributes:
+
+Property | Description
+---------|-------------
+Timestamp | Time and date of the event in UTC.
+`rt` | Milliseconds since Unix epoch.
+`src` | The IP address of the request originator.
+`org_id` | The originating organization ID.
```

Comment:
suggestion: although we keep `org_id` for our internal records, we don't need to expose this to the customer, since they can only get their own logs. @cecilmoore @haydenlam wdyt?

Reply:
I'd be inclined to keep `org_id` since there is a chance that a customer is running more than one Konnect organization. Not for Dev/Prod, but for staging config in their "Prod" organization such as setting up Audit Logs or SSO Testing or multi Geo which are not segregated by runtime groups.

And secondly, if these logs are ever used to bring up issues back to us, we would certainly appreciate the org_id being unambiguously a part of the logs.

Are there technical or cost reasons we'd want to avoid this?